common/models/templates/src/tokenization.ts

```diff
@@ -22,21 +28,66 @@ export interface Tokenization {
   caretSplitsToken: boolean
 }
 
-export function tokenize(wordBreaker: WordBreakingFunction, context?: Partial<Context>): Tokenization {
+export function tokenize(
+  wordBreaker: WordBreakingFunction,
+  context: Partial<Context>,
+  options?: {
+    /** Characters to rejoin to preceding tokens if found immediately pre-caret. */
+    rejoins?: string[]
+  }
+): Tokenization {
+  const rejoins = options?.rejoins || ["'"];
```

Comment:
Why the single quote character? Could you add a comment?

Reply:
Comment has been added.  Let me know if I should alter / improve it in some manner.